controllers/articles-controller.js

```diff
@@ -1,9 +1,21 @@
+const { sort } = require("../db/data/test-data/articles");
 const {
   selectArticles,
   selectAllArticles,
   updateArticleVotes,
 } = require("../models/articles-model");
 
+const validSortBy = [
+  "author",
+  "title",
+  "article_id",
+  "topic",
+  "created_at",
+  "votes",
+  "comment_count",
+];
```

Comment:
While it is highly unlikely that the columns will change, a good challenge for you would be to find out how you can get the valid columns programmatically without hard coding. _Hint it only takes one google search as it is a pretty standard psql query that we haven't taught_

Reply:
<img width="641" alt="Screenshot 2024-08-27 at 14 45 36" src="https://github.com/user-attachments/assets/5d0f3dee-1f90-41d9-b9a3-5661e744c5a3">


Is it that?